.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 14
```

Comment:
github action probably has matrix that supports multiple java versions, though i don't know how to do such

Reply:
Yes, I does. I will add j8 as well.